docs/source/tutorials/psij.ipynb

```diff
@@ -127,28 +127,20 @@
     "from psij import JobState\n",
     "\n",
     "executable = \"/bin/date\"\n",
-    "ex = JobExecutor.get_instance(environment)\n",
+    "ex = JobExecutor.get_instance(executor_type)\n",
```

Comment:
I'm not quite sure how this works, but if this line sees `executor_type` defined earlier, there is no need to instantiate a new executor since you could simply use `ex` from before.